src/azure-cli/azure/cli/command_modules/util/tests/latest/test_rest.py

```diff
@@ -13,14 +14,19 @@ class ResourceGroupScenarioTest(ScenarioTest):
 
     @ResourceGroupPreparer(name_prefix='cli_test_rest')
     def test_rest(self, resource_group):
+        curr_dir = os.path.dirname(os.path.realpath(__file__))
         self.kwargs.update({
-            'sa': self.create_random_name("tmpst", length=10)
+            'sa': self.create_random_name("tmpst", length=10),
+            'storage_account_create_body': os.path.join(curr_dir,
+                                                        'rest_storage_account_create_body.json').replace('\\', '\\\\'),
```

Comment:
Maybe we can use the same mechanism of `TEST_DIR` from keyvault module?

https://github.com/Azure/azure-cli/blob/a8c1a56758dfa9c4d862e6ed46dd50080eff8391/src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py#L31

(Though let's use `os.path.dirname` instead of `..`)

Reply:
OK, this way is more readable. I've modified it